src/test/java/SixPMTest.java

```diff
@@ -0,0 +1,102 @@
+        WebElement accessoriesButton = driver.findElement(accessoriesButtonLoc);
+        wait.until(ExpectedConditions.elementToBeClickable(accessoriesButtonLoc));
+        actions.moveToElement(accessoriesButton).perform();
+
+        By aviatorLoc = By.xpath("//*[text()=\"Aviators\"]");
+        WebElement aviatorButton = driver.findElement(aviatorLoc);
+        wait.until(ExpectedConditions.elementToBeClickable(aviatorButton));
+
+        actions.moveToElement(aviatorButton).click().build().perform();
+
+
+        By searchPageLoc = By.xpath("//*[@class='py-z ns-z']");
+        wait.until(ExpectedConditions.visibilityOfAllElementsLocatedBy(searchPageLoc));
+
+        List<WebElement> aviators = driver.findElements(By.xpath("//*[@class='py-z ns-z']"));
+        WebElement aviator = Utils.randomItem(aviators);
+
+        actions.moveToElement(aviator).click().build().perform();
+
+        By itemPageLoc = By.xpath("//*[@class='Gb-z']");
+        wait.until(ExpectedConditions.elementToBeClickable(itemPageLoc));
+
+        String expectedName = driver.findElement(By.xpath("//*[@class='fN-z']")).getAttribute("innerHTML");
+        String expectedPrice = driver.findElement(By.xpath("//*[@class='tK-z yK-z']")).getAttribute("aria-label");
+
+        WebElement addToShoppingBagButton = driver.findElement(By.xpath("//button[text()='Add to Shopping Bag']"));
+        actions.moveToElement(addToShoppingBagButton).click().build().perform();
+
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[@class='oz-z']")));
+
+        WebElement viewBagButton = driver.findElement(By.xpath("//*[@class='sz-z']"));
+        actions.moveToElement(viewBagButton).click().build().perform();
+
+        By itemInBagContainerLoc = By.xpath("//*[@class='pc-z']");
+        wait.until(ExpectedConditions.visibilityOfAllElementsLocatedBy(itemInBagContainerLoc));
+
+        String actualName = driver.findElement(By.xpath("//div[@class='Wk-z']/a")).getText().replace('\n', ' ');
+
+        String actualPrice = driver.findElement(By.xpath("//*[@class='yh-z Bh-z']")).getText();
+
+        SoftAssert softAssert = new SoftAssert();
+        softAssert.assertEquals(actualName, expectedName, "Name of the item doesn't match the expected value.");
+        softAssert.assertEquals(actualPrice, expectedPrice, "Price of the item doesn't match the expected value.");
+        softAssert.assertAll();
+
+        WebElement quantityButton = driver.findElement(By.name("quantity"));
+        actions.moveToElement(quantityButton).click().build().perform();
+
+        WebElement removeOption = driver.findElement(By.xpath("//*[text()='Remove']"));
+        removeOption.click();
+
+        String textCheck = "Nothing to see here yet! Sign in to see items that you've previously placed in your Bag or check out all the awesome things you can buy on 6pm!";
+        wait.until(ExpectedConditions.textToBe(By.xpath("//h1/following-sibling::p"), textCheck));
```

Comment:
No need to have this text, it's error prone.
Just check that the product gets invisible in shopping bag after removal.

Reply:
just to ensure that there is no such item after its removal?